src/item/mutations.ts

```diff
@@ -88,7 +83,8 @@ export default (queryConfig: QueryClientConfig) => {
 
     // get parent key
     const childrenKey = !parentId
-      ? OWN_ITEMS_KEY
+      ? // todo: this should be removed when we remove this function, previously was OWN_ITEMS
+        ['']
```

Comment:
Should we create an issue for that (if its not already the case)?

Reply:
I do not fully graasp what this function is supposed to do. I think it has something to do with the optimistic updates. Ideally we should be able to remove the function as a whole. But I am not sure how feasible that is. @pyphilia